package/docker/Dockerfile.ubuntu-jammy

```
@@ -18,16 +18,9 @@ RUN    apt-get update
     && apt-get upgrade --yes                                                            \
     && apt-get install --yes --no-install-recommends /kframework_amd64_ubuntu_jammy.deb
 
-ARG UV_VERSION
-RUN    curl -LsSf https://astral.sh/uv/$UV_VERSION/install.sh | sh \
-    && uv --version
-
-COPY pyk /pyk
-RUN pipx ensurepath           \
-    && . /root/.profile       \
-    && cd /pyk                \
-    && make build             \
-    && pip install dist/*.whl \
-    && rm -rf /pyk
-
-RUN rm -rf /kframework_amd64_ubuntu_jammy.deb
+COPY kframework-*.whl ./
+RUN    pipx ensurepath                              \
+    && . /root/.profile                          \
+    && pip install /kframework-*.whl             \
+    && rm -rf /kframework-*                      \
+    && rm -rf /kframework_amd64_ubuntu_jammy.deb
```

Wildcards can probably be eliminated here. Also, `-rf` is no longer needed here I think.

```suggestion
COPY kframework.whl /kframework.whl
RUN    pipx ensurepath             \
    && . /root/.profile            \
    && pip install /kframework.whl \
    && rm /kframework.whl          \
    && rm /kframework_amd64_ubuntu_jammy.deb
```

(It'd be even better to remove the `.deb` as the last step of the previous `RUN`.)

---

I removed the `-rf` and moved the `.deb` to prior installing the wheel.

I tried out the installation of a `kframework.whl` wheel before, but pip somehow requires the filename to include the version of the wheel. Thus, I left the filename as is and use wildcards accordingly.